src/gui/tray/UserModel.cpp

```diff
@@ -646,11 +651,12 @@ Q_INVOKABLE void UserModel::openCurrentAccountTalk()
     if (_users.isEmpty())
         return;
 
-    QString url = _users[_currentUserId]->server(false) + "/apps/spreed";
-    if (!(url.contains("http://") || url.contains("https://"))) {
-        url = "https://" + _users[_currentUserId]->server(false) + "/apps/spreed";
+    auto talkApp = _users[_currentUserId]->talkApp();
+    if(talkApp){
```

Comment:
Missing space after if and before opening curly brace, should be: `if (talkApp) {`